docs/aws.rst

```diff
@@ -16,68 +84,62 @@ API
 S3
 ***
 
+.. _aws_s3:
+
 ========
 Overview
 ========
 
-S3 is Amazon Web Service's object storage service that allows users to store and access data objects. The Parson's class is a high level wrapper of the AWS SDK `boto3 <https://boto3.amazonaws.com/v1/documentation/api/latest/index.html>`_. It allows users to upload and download files from S3 as well as manipulate buckets.
+The ``S3`` class allows interaction with Amazon Web Service's `object storage service <https://aws.amazon.com/s3/>`_
+to store and access data objects. It is a wrapper around the AWS SDK `boto3 <https://boto3.amazonaws.com/v1/documentation/api/latest/index.html>`_.
+It provides methods to upload and download files from S3 as well as manipulate buckets.
 
 .. note::
   Authentication
-    Access to S3 is controlled through AWS Identity and Access Management (IAM) users in the `AWS Managerment Console <https://aws.amazon.com/console/>`_ . Users can be granted granular access to AWS resources, including S3. IAM users are provisioned keys, which are required to access the S3 class. 
+    Access to S3 is controlled through AWS Identity and Access Management (IAM) users in the `AWS Managerment Console <https://aws.amazon.com/console/>`_ .
+    Users can be granted granular access to AWS resources, including S3. IAM users are provisioned keys, which are required to access the S3 class.
 
 ==========
 QuickStart
 ==========
 
-Instantiate class with credentials.
+S3 credentials can either be passed as environmental variables (``AWS_ACCESS_KEY_ID`` and ``AWS_SECRET_ACCESS_KEY``) or
+as keyword arguments.
 
 .. code-block:: python
 
    from parsons import S3
 
-   # First approach: Use API credentials via environmental variables
+   # First approach: Pass API credentials via environmental variables
    s3 = S3()
 
    # Second approach: Pass API credentials as arguments
    s3 = S3(aws_access_key_id='MY_KEY', aws_secret_access_key='MY_SECRET')
 
-   # Third approach: Use credentials stored in AWS CLI file ~/.aws/credentials
-   s3 = S3()
```

Comment:
Is there a reason that you removed this option?

Reply:
I thought it seemed redundant to have the line `s3 = S3()` twice. What if I add a comment to the first line that `s3 = S3()` works with either env variables or the AWS CLI file option?

Reply:
Seems like a good idea. I'd just suggest clarifying in line 106.